projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -45,7 +45,20 @@ function getCartItem(name: string) {
   return cy.get('cx-cart-item-list').contains('cx-cart-item', name);
 }
 
-function goToProductFromSearch(id: string, mobile: boolean) {
+function checkCartSummary(subtotal: string) {
+  cy.get('cx-order-summary').within(() => {
+    cy.get('.cx-summary-row:first').contains('Subtotal');
+    cy.get('.cx-summary-amount').should('contain', subtotal);
+  });
```

Comment:
Give first general feedback fast (important for changes in architecture, big features). Often PR author will work on another issue relying on this branch. Changing things sooner will save some time.